ui/app/src/modules/Preview/Tools/AudiencesPanel.tsx

```diff
@@ -96,25 +101,27 @@ export function AudiencesPanelUI(props: AudiencesPanelUIProps) {
   const classes = useStyles({});
   const {
     audiencesResource,
+    profile,
+    profileApplying,
+    profileApplied,
     onFormChange,
     saveProfile,
     setDefaults
   } = props;
-  const audiencesData = audiencesResource.read();
-  const { config, profile } = audiencesData;
+  const config = audiencesResource.read();
 
   return (
     <ToolPanel title={translations.audiencesPanel}>
       {
         <>
           <Grid className={classes.PanelMargin}>
             {
-              Object.keys(config.fields).map((property: any, index: number) => (
-                <div key={index}>
+              Object.keys(config.fields).map((field: any) => (
+                <div key={field}>
```

Comment:
What’s this div for? If it’s just for wrapping the two elements with no structural or stylistic purpose, use a fragment